Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Tabular/Booking/TripLocation.hs

```diff
@@ -18,17 +18,17 @@
 {-# LANGUAGE TemplateHaskell #-}
 {-# OPTIONS_GHC -Wno-orphans #-}
 
-module Storage.Tabular.Booking.BookingLocation where
+module Storage.Tabular.Booking.TripLocation where
 
-import qualified Domain.Types.Booking.BookingLocation as Domain
+import qualified Domain.Types.Booking.TripLocation as Domain
 import Kernel.Prelude
 import Kernel.Storage.Esqueleto
 import Kernel.Types.Id
 
 mkPersist
   defaultSqlSettings
   [defaultQQ|
-    BookingLocationT sql=booking_location
```

Comment:
Why change everything including type names lol? just change table pointer, you didn't change anything else

Reply:
But isn't it better to have correct names in code also 
cc : @hkmangla @NikithShetty

Reply:
@DonDokidon in this feature, `ride` table will have actual from and to location of the trip. So it would need table similar to `BookingLocation` as well. Instead of creating a new one, we thought its better to share the location table to save space as well. Hence the rename to `TripLocation` as it will be referenced by both Booking and Ride tables

Reply:
@NikithShetty we don't have any link to from/ToLocation in Ride, why you need to reference it from Ride? Do you want to add it? 
We will need to add bunch of tables and fields like search_request_2 to support backward compatibility, rename everything everytime you do that just counterproductive and troublesome for other people, since they'll need to rebase over it/remember new names.